.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/BatchTests.cs

```diff
@@ -56,15 +56,13 @@ public async Task SimpleBatchCompletionsTest()
             }
         }.ToBinaryContent();
 
-        ClientResult response = await batchClient.CreateBatchAsync(requestContent);
-        BatchObject batchObj = ExtractAndValidateBatchObj(response);
+        CreateBatchOperation operation = await batchClient.CreateBatchAsync(requestContent, true);
 
         // Poll until we've completed, failed, or were canceled
-        while ("completed" != batchObj.Status)
-        {
-            response = await batchClient.GetBatchAsync(batchObj.Id, new());
-            batchObj = ExtractAndValidateBatchObj(response);
-        }
+        operation.WaitForCompletion();
+
+        ClientResult response = operation.GetBatch(null);
+        BatchObject batchObj = ExtractAndValidateBatchObj(response);
 
         Assert.That(batchObj.OutputFileID, Is.Not.Null.Or.Empty);
```

Comment:
Should we validate correctness of the LRO implementation?  i.e. HasCompleted is true after `WaitForCompletion` returns, maybe also that manually calling `UpdateStatus` in a loop is equivalent to `WaitForCompletion`?  If the latter isn't present in the OAI test suite and the AOAI subtype doesn't make any changes to the UpdateStatus implementation, maybe we add that test there instead.

Reply:
noticed this at the top of the test "[Ignore("Azure OpenAI does not yet support batch file uploads")]" so this test is ignored for now and will need to be reworked when it is supported